src/rebar_utils.erl

```diff
@@ -401,11 +400,6 @@ compare({Priority, A}, {Secondary, B}) when not is_tuple(A), is_tuple(B) ->
        A  > KB -> {Priority, B, A}
     end.
 
-%% Implements wc -l functionality used to determine patchcount from git output
-line_count(PatchLines) ->
-    Tokenized = rebar_string:lexemes(PatchLines, "\n"),
-    {ok, length(Tokenized)}.
-
```

Comment:
I would avoid dropping code like that. Experience tells us that plugins might rely on "internal exports" like that and we could be creating a backwards-incompatible release.

Reply:
Ok, I will resurrect it.